Sources/Orbit/Support/Forms/InputContent.swift

```diff
@@ -1,30 +1,35 @@
 import SwiftUI
 
 /// Content for inputs that share common layout with a prefix and suffix.
-struct InputContent<Content: View, Prefix: View, Suffix: View>: View {
+public struct InputContent<Content: View, Prefix: View, Suffix: View>: View {
 
     @Environment(\.idealSize) private var idealSize
     @Environment(\.isEnabled) private var isEnabled
     @Environment(\.textColor) private var textColor
 
     public let verticalPadding: CGFloat = .small // = 44 @ normal text size
 
-    let state: InputState
-    let message: Message?
-    let isPressed: Bool
-    let isEditing: Bool
-    @ViewBuilder let content: Content
-    @ViewBuilder var prefix: Prefix
-    @ViewBuilder var suffix: Suffix
+    private let label: String
+    private let state: InputState
+    private let message: Message?
+    private let isPressed: Bool
+    private let isEditing: Bool
+    private let isPlaceholder: Bool
+    @ViewBuilder private let content: Content
+    @ViewBuilder private let prefix: Prefix
+    @ViewBuilder private let suffix: Suffix
```

Comment:
Should these still be `@ViewBuilder` if they are called in the inits with `()`? By the time they are assigned, they should be resolved values, shouldn't they?

Reply:
The `@ViewBuilder` probably does not need to be specified, but it may be better to keep it for semantics (same reason we do it for `@ViewBuilder var` properties and functions) and to match the init parameters.